lib/frontend/admin/pages/article/article_add.dart

```diff
@@ -166,27 +168,38 @@ class _AddBlogPostState extends State<AddBlogPost> {
               isEnabled: isEnabled,
               tags: tagsController.text.split(','),
               author: authorController.text,
-              content: await controller.getText(),
+              content: await quillEditorcontroller.getText(),
```

Comment:
There is no need to throw an exception. When the user clicks the save button, they will be prompted to either fill out the entire form or upload an image. This will prevent any errors from occurring.